13/umbraco-cms/fundamentals/backoffice/property-editors/built-in-umbraco-property-editors/media-picker-3.md

```diff
@@ -83,6 +86,23 @@ Global crops are configured on the Image Cropper property of the Image Media Typ
 }
 ```
 
+### Multiple enabled without Modelsbuilder to retrieve IEnumerable<IPublishedContent> data
```

Comment:
Shouldn't it be after the "Multiple enabled without Modelsbuilder"?
Or, perhaps it should even be a sub section below that one, as it's still multiple enabled and without the Models Builder 🤔